From 2e250c1db1828afb543219b941edf876687c1182 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Mon, 21 May 2018 11:30:58 +0800
Subject: [PATCH 3858/5242] MLK-18355-9: dts: Add hdmi rx cec property

commit  e4ea311343f3b346c6bd985f26be8b89f4522c32 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add hdmi rx cec property to imx8qm HDMI RX driver.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts     |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
index 08a3c30..a502793 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
@@ -133,6 +133,7 @@
 };
 
 &hdmi_rx {
+	fsl,cec;
 	assigned-clocks = <&clk IMX8QM_HDMI_RX_HD_REF_SEL>,
 						<&clk IMX8QM_HDMI_RX_PXL_SEL>,
 						<&clk IMX8QM_HDMI_RX_HD_REF_DIV>;
-- 
1.7.9.5

